src/Visualization/Python/PlotPowerMonitors.py

```diff
@@ -299,6 +309,7 @@ def plot_power_monitors(
         "'--elements' / '-e' patterns."
     ),
 )
+@click.option("--figsize", nargs=2, type=float, help="Figure size in inches.")
```

Comment:
Test the new option?

Reply:
We don't currently have a way to test plot layouts

Reply:
Sorry my bad, I wasn't clear. I meant test that passing in a value that isn't the default doesn't cause an error.

Reply:
ok sure